stacks/SiriGeneratorStack.ts

```diff
@@ -95,4 +29,10 @@ export function SiriGeneratorStack({ stack }: StackContext) {
         siriSXUnvalidatedBucket.bucketName,
     );
     validatorBucket.addEventNotification(EventType.OBJECT_CREATED, new LambdaDestination(siriValidator));
+
+    createSiriApi(stack, siriSXBucket);
```

Comment:
It would be nice here if we could split the validator and api out into their own stacks rather than having everything in the generator stack

Reply:
done